prestodb/cdh5-hive-kerberized/Dockerfile

```diff
@@ -70,8 +70,9 @@ RUN keytool -genkeypair \
     -keystore /etc/presto/conf/keystore.jks \
     -keypass password \
     -storepass password \
-    -dname "CN=presto-master, OU=, O=, L=, S=, C="
-RUN chmod 644 /etc/presto/conf/keystore.jks
```

Comment:
The change to how chmod is invoked seems unrelated. Should it be done in a separate commit?